packages/devtools_app/test/test_infra/utils/file_utils.dart

```diff
@@ -33,11 +33,9 @@ Future<String> resolvePackagePath(String package) async {
   // so the packages directory is the current directory's grandparent.
   // TODO(https://github.com/flutter/flutter/issues/20907): Remove the workaround here.
   String packagesPath;
-  if (io.Directory.current.path.endsWith('test')) {
-    packagesPath = io.Directory.current.parent.parent.path;
-  } else {
-    packagesPath = io.Directory.current.parent.path;
-  }
+  packagesPath = io.Directory.current.path.endsWith('test')
```

Comment:
I wonder why the duplicated code lint isn't catching that `io.Directory.current` and `io.Directory.current.parent` are duplicated

Reply:
I tuned it up to 3 dupes to reduce the # of false positives. Can revisit with 2 dupes after the initial cleanup lands.